src/test/java/seedu/address/logic/commands/applicant/AddApplicantCommandTest.java

```diff
@@ -168,6 +168,26 @@ public void addInterview(Interview interview) {
             throw new AssertionError("This method should not be called.");
         }
 
+        @Override
```

Comment:
Personally, I don't agree with having passInterview, failInterview, etc, as methods in Model. I feel that there is an overlap in logic with setInterview. 

The logic for passing an interview can be broken down into:
1. Retrieving Interview from the ModelManager
2. Marking that Interview as passed
3. Setting the old interview in ModelManager to the newly passed interview.